python/lsst/ts/MTAOS/model.py

```diff
@@ -87,10 +137,34 @@ def __init__(self, config, ofc_data, log=None):
         else:
             self.log = log.getChild(type(self).__name__)
 
-        # Configuration
-        self.config = config
+        self.instrument = instrument
+        self.data_path = data_path
+        self.run_name = run_name
+
+        self.collections = collections
+        self.pipeline_instrument = (
+            pipeline_instrument
+            if pipeline_instrument is not None
+            else dict(
+                comcam="lsst.obs.lsst.LsstComCam",
+                lsstCam="lsst.obs.lsst.LsstCam",
+                lsstFamCam="lsst.obs.lsst.LsstCam",
+            )
+        )
+        self.pipeline_n_processes = pipeline_n_processes
+        self.data_instrument_name = (
+            data_instrument_name
+            if data_instrument_name is not None
+            else dict(
+                comcam="LSSTComCam",
+                lsstCam="LSSTCam",
+                lsstFamCam="LSSTCam",
+            )
+        )
+        self.zernike_table_name = zernike_table_name
+        self.reference_detector = reference_detector
 
-        self.camera = CamFactory.createCam(self.config.getCamType())
```

Comment:
I think you may need to remove the doc string of this attribute. You do not hold it anymore.